content/hi/case-studies/appdirect/index.html

```diff
@@ -0,0 +1,84 @@
+
+<h2>प्रभाव</h2>
+
+<p>कुबेरनेट्स प्लेटफ़ॉर्म ने पिछले कुछ वर्षों में इंजीनियरिंग टीम की 10 गुना वृद्धि में सहायता की है। इस तथ्य के साथ कि वे लगातार नई सुविधाएँ जोड़ रहे थे, लैकर्टे कहते हैं, "मुझे लगता है कि अगर हमारे पास यह नया बुनियादी ढांचा नहीं होता तो हमारी गति बहुत धीमी हो जाती।" कुबेरनेट्स और सेवाओं में जाने का मतलब है कि एससीपी कमांड के साथ कस्टम-निर्मित, भंगुर शेल स्क्रिप्ट पर कम निर्भरता के कारण तैनाती बहुत तेज हो गई है। नए संस्करण को तैनात करने का समय 4 घंटे से घटकर कुछ मिनट रह गया है। इसके अतिरिक्त, कंपनी ने डेवलपर्स के लिए चीजों को स्व-सेवा बनाने के लिए बहुत प्रयास किया। लैकर्टे कहते हैं, "नई सेवा में शामिल होने के लिए <a href="https://www.atlassian.com/software/jira">जिरा</a> टिकट या तीन अलग-अलग टीमों के साथ बैठक की आवश्यकता नहीं होती है।" आज, कंपनी प्रति सप्ताह 1,600 तैनाती देखती है, जबकि पहले यह संख्या 1-30 थी। कंपनी ने अपने मार्केटप्लेस और बिलिंग मोनोलिथ को पुराने EC2 होस्ट से कुबेरनेट्स में स्थानांतरित करने के साथ-साथ ऑटोस्केलिंग का लाभ उठाकर लागत बचत भी हासिल की, क्योंकि व्यावसायिक घंटों के दौरान ट्रैफ़िक अधिक होता है।</p>
+
+{{< case-studies/quote author="Alexandre Gervais, Staff Software Developer, AppDirect" >}}
+"यह इंजीनियरिंग संस्कृति में एक बड़ा बदलाव था, लेकिन पैमाने और गति के मामले में इसके लाभ निर्विवाद हैं।"
+{{< /case-studies/quote >}}
+
+{{< case-studies/lead >}}
+क्लाउड-आधारित उत्पादों और सेवाओं के लिए अपने एंड-टू-एंड कॉमर्स प्लेटफॉर्म के साथ, <a href='https://www.appdirect.com/'>AppDirect</a> कॉमकास्ट और GoDaddy जैसे संगठनों को सरल बनाने में मदद कर रहा है। 2009 से डिजिटल आपूर्ति श्रृंखला।
+{{< /case-studies/lead >}}
+
+<p>जब सॉफ्टवेयर डेवलपमेंट के निदेशक पियरे-एलेक्जेंडर लैकरटे ने 2014 में वहां काम करना शुरू किया, तो कंपनी के पास "टॉमकैट इंफ्रास्ट्रक्चर पर एक मोनोलिथ एप्लिकेशन तैनात था, और इसे जो होना चाहिए उसके लिए पूरी रिलीज प्रक्रिया जटिल थी," वे कहते हैं। "इसमें बहुत सारे मैन्युअल चरण शामिल थे, जिसमें एक इंजीनियर एक फीचर बनाता था फिर एक पुल अनुरोध बनाता था, और एक क्यूए या कोई अन्य इंजीनियर फीचर को मान्य करता था। फिर इसे विलय कर दिया जाता है और कोई और तैनाती का ध्यान रखेगा। इसलिए हमारे पास बाधाएं थीं किसी सुविधा को उत्पादन के लिए भेजने की प्रक्रिया में है।"</p>
+
+<p>उसी समय, 40 की इंजीनियरिंग टीम बढ़ रही थी, और कंपनी अपने उत्पादों में अधिक से अधिक सुविधाएँ जोड़ना चाहती थी। प्लेटफ़ॉर्म टीम के सदस्य के रूप में, लैकर्टे ने कई टीमों से सुनना शुरू किया जो <a href="https://nodejs.org/">Node.js</a> से लेकर विभिन्न फ़्रेमवर्क और भाषाओं का उपयोग करके एप्लिकेशन को तैनात करना चाहते थे। a href='http://spring.io/projects/spring-boot'>Spring boot JAVA</a>। उन्हें जल्द ही एहसास हुआ कि विकास को समर्थन देने और गति बढ़ाने के लिए, कंपनी को एक बेहतर बुनियादी ढांचे और एक ऐसी प्रणाली की आवश्यकता है जिसमें टीमें स्वायत्त हों, अपनी तैनाती स्वयं कर सकें और उत्पादन में अपनी सेवाओं के लिए जिम्मेदार हों।</p>

```

Comment:
instead of using तैनात use डेप्लॉय as it will be easy to understand check for this in entire page.